# Adds the trknotify dependency for macOS which isn't included for some reason

--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -1180,6 +1180,7 @@ if (is_win) {
     ]
 
     deps = [
+      "//iridium:trknotify",
       ":browser_dependencies",
       ":child_dependencies",
       "//chrome/app:command_ids",
